lib/jekyll-admin/directory.rb

```diff
@@ -0,0 +1,58 @@
+module JekyllAdmin
+  class Directory
+    extend Forwardable
+    def_delegator :@path, :basename, :name
+    def_delegator :@path, :mtime, :modified_time
+    attr_reader :path, :splat, :content_type, :base
+
+    include Enumerable
+    include JekyllAdmin::URLable
+    include JekyllAdmin::APIable
+
+    TYPE = "directory".freeze
```

Comment:
It's a strange ruby concept, but strings that aren't modified should be stored as `:symbols` which are treated differently for garbage collection purposes.